content/2022/robot-valjak.md

```diff
@@ -2,3 +2,212 @@
 title: Robot valjak
 summary: Robot u obliku valjka je projekat rađen na letnjem kampu za stare polaznike 2022. godine od Nikole Ristanovića i Filipa Bajraktarija.
 ---
+
+![Grafički apstrakt](/images/2022/robot-valjak/graficki_apstrakt.svg)
```

Comment:
Izbacite PNG slike koje su zamenjene SVG verzijama iz repozitorijuma, kao što je, između ostalog, `Graficki apstrakt.png`.